test/conftest.py

```diff
@@ -0,0 +1,62 @@
+import os
+import pytest
+from pathlib import Path
+
+
+@pytest.fixture(scope="session")
+def output_path(request):
+    """Set the output path: This contains control and lab directories for each
+    test and test output files - e.g. CHECKSUMS
+    """
+    path = request.config.getoption('--output-path')
+    if path is None:
+        # Set default to /scratch/PROJECT/USER/test-model-repro/
+        project = os.environ.get('PROJECT')
+        user = os.environ.get('USER')
+        path = f'/scratch/{project}/{user}/test-model-repro'
+    return Path(path)
+
+
+@pytest.fixture(scope="session")
+def control_path(request):
+    """Set the path of the model configuration directory to test"""
+    path = request.config.getoption('--control-path')
+    if path is None:
+        # Set default to current working directory
+        path = Path.cwd()
+    return Path(path)
+
+
+@pytest.fixture(scope="session")
+def checksum_path(request, control_path):
+    """Set the path of the model configuration directory to test"""
+    path = request.config.getoption('--checksum-path')
+    if path is None:
+        # Set default to checksum stored on model configuration
+        path = control_path / 'testing' / 'checksum' / 'CHECKSUM'
```

Comment:
Will this be in the form `testing/checksum/CHECKSUM` anymore, or will it be like `testing/checksum/access-om2-checksums-v1.0.yaml` like in the `./test/test/resources/` directory? I can't remember what we'd decided on regarding the actual filename :D

Reply:
Updated to `historic-3hr-checksum.yaml`